docs/devices/0402946.md

```diff
@@ -43,123 +60,19 @@ To control this lock publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set`
 To read the current state of this lock publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"state": ""}`.
 This lock exposes a lock state which can be found in the published state under the `lock_state` property. It's not possible to read (`/get`) or write (`/set`) this value. The possible values are: `not_fully_locked`, `locked`, `unlocked`.
 
-### Battery (numeric)
```

Comment:
Don't remove these, this part is automatically generated based on the `exposes`.